### PR TITLE
[mapbox-gl] Update for v1.3.0

### PR DESCRIPTION
**Pull request by danvk**
Also updates for v1.4.0 since there were no API changes in that version.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mapbox/mapbox-gl-js/releases
  - `visualizePitch`: https://github.com/mapbox/mapbox-gl-js/pull/8296/files
  - `validate` option for `queryRenderedFeatures` / `querySourceFeatures`: https://github.com/mapbox/mapbox-gl-js/pull/8211/files
  - `Popup.trackPointer`: https://github.com/mapbox/mapbox-gl-js/pull/7786/files
  - `Popup.getElement`: https://github.com/mapbox/mapbox-gl-js/pull/8123/files
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/mapbox-gl/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Mapbox GL JS v0.54.0
+// Type definitions for Mapbox GL JS v1.3.0
```
Comment (dmytro-gokun) [inline types/mapbox-gl/index.d.ts]: The PR title says it's for 1.4.0... Confusing.

**Review by dmytro-gokun — COMMENTED** [review]
So... does this PR contains ALL the API differences there are between 0.54.0 and 1.4.0 or only some? The reason i ask is that it's quite a version bump, we need to make sure we do not break a lot of peoples code..

File: types/mapbox-gl/index.d.ts (hunk 1 above)
Comment (danvk) [reply types/mapbox-gl/index.d.ts]: Yes, this is a bit confusing. There were no API changes in [v1.4.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.4.0) so these typings will work for v1.3.0 or v1.4.0. I can change the PR description if you think it makes more sense.

**Comment by danvk** [comment]
Yes, this contains all the API changes between v0.54.0 and v1.4.0. I went through the release notes and found all PRs that affected the API.

It's hard for me to see how any of these changes could break a user since there are only additions of methods and parameters.

File: types/mapbox-gl/index.d.ts (hunk 1 above)
Comment (dmytro-gokun) [reply types/mapbox-gl/index.d.ts]: Yes, plz do

**Comment by dmytro-gokun** [comment]
That's really great! Thanks a lot for the job you've done here!

**Review by dmytro-gokun — APPROVED** [review]

**Comment by dmytro-gokun** [comment]
@danvk Do you mind resolving the merge conflict? Thanks.

**Comment by danvk** [comment]
@dmytro-gokun done

**Review by dmytro-gokun — APPROVED** [review]

**Pull request merged**